app/src/components/TipsAction.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts">
+import { useTipsStore } from '@/stores/tips'
+import { storeToRefs } from 'pinia'
+import LoadingButton from './LoadingButton.vue'
+import type { AddressLike } from 'ethers'
+
+const tipStore = useTipsStore()
+const { pushTip, sendTip } = useTipsStore()
+const { totalTipAmount, sendTipLoading, pushTipLoading } = storeToRefs(tipStore)
```

Comment:
For latter: TO Be refactor. 
A component should only disply things. 
No processing, no call of store.